collectd/collectd.go

```diff
@@ -0,0 +1,216 @@
+package collectd
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"net"
+	"sync"
+	"time"
+
+	"github.com/kimor79/gollectd"
+)
+
+// DefaultPort for colletd is 25826
```

Comment:
It may be useful having the package export what the default is, for documentation purposes etc.

Scrap my comment about not having a comment. It was a poor review comment on my part.